zh_cn.lang

```diff
@@ -35,43 +35,43 @@ gui.xaero_numeral=数值
 gui.xaero_both=两者
 gui.xaero_quick_eat=快速进食
 gui.xaero_toggle_sneak=切换潜行
-gui.xaero_quick_use_1=快速使用 1
-gui.xaero_quick_use_2=快速使用 2
-gui.xaero_quick_use_3=快速使用 3
-gui.xaero_quick_use_4=快速使用 4
-gui.xaero_quick_use_5=快速使用 5
-gui.xaero_quick_use_6=快速使用 6
-gui.xaero_quick_use_7=快速使用 7
-gui.xaero_quick_use_8=快速使用 8
-gui.xaero_quick_use_9=快速使用 9
-gui.xaero_restores=还原
-gui.xaero_hunger_low=饱食度低
-gui.xaero_hp_low=健康值低
+gui.xaero_quick_use_1=快速使用1
+gui.xaero_quick_use_2=快速使用2
+gui.xaero_quick_use_3=快速使用3
+gui.xaero_quick_use_4=快速使用4
+gui.xaero_quick_use_5=快速使用5
+gui.xaero_quick_use_6=快速使用6
+gui.xaero_quick_use_7=快速使用7
+gui.xaero_quick_use_8=快速使用8
+gui.xaero_quick_use_9=快速使用9
+gui.xaero_restores=恢复值
+gui.xaero_hunger_low=低饥饿值
+gui.xaero_hp_low=低生命值
 gui.xaero_potion_status_settings=药水状态设置
 gui.xaero_potion_names=药水效果名称
-gui.xaero_air_setting=氧气量提醒
-gui.xaero_air_low=氧气量低
+gui.xaero_air_setting=氧气值提醒
+gui.xaero_air_low=低氧气值
 gui.xaero_entity_info=实体信息
-gui.xaero_box_entity_info=An interface displaying information about the living entity that you are pointing at, including players.
+gui.xaero_box_entity_info=显示你所指向的生物实体的信息，包括玩家。
 gui.xaero_entity_info_settings=实体信息设置
-gui.xaero_entity_info_stay=继续显示实体信息
-gui.xaero_entity_info_distance=可达距离
-gui.xaero_entity_info_max_hearts=Max Heart Number
-gui.xaero_entity_info_numbers=Health in Numbers
-gui.xaero_show_full_amount=Full Held Item Quantity
+gui.xaero_entity_info_stay=继续显示
+gui.xaero_entity_info_distance=触及距离
+gui.xaero_entity_info_max_hearts=最大生命值
```

Comment:
This is supposed to say "hearts" specifically, not health. If an entity has 100 hearts of health and this is set to 10, it'll display 10 hearts. At 50/100 real hearts, it'll display 5/10 hearts.

Reply:
OK, I will bring more information in the translation